dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -225,6 +225,7 @@
     E4311( "SQL query contains illegal keywords" ),
     E4312( "Current user is not authorised to read data from SQL view: `{0}`" ),
     E4313( "SQL query contains variable names that are invalid: `{0}`" ),
+    E4314( "List of `{0}` should contain all elements of list `{1}` when relationship entity is `{2}`." ),
```

Comment:
Could we make this more understandable for end users?

Maybe "Provided attributes/dataelements (uids) are not part of the selected tet/program/programstage (uids)", written out in a proper text of course :) Right now I think it might be confusing

Reply:
resolved